test/functional/inspec_init_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'functional/helper'
+require 'fileutils'
+require 'tmpdir'
+
+describe 'inspec init' do
+  include FunctionalHelper
+
+  tmpdir = Dir.tmpdir
+  puts tmpdir
+
+  describe 'inspec init profile with/slash' do
+    it 'prevents profile names without valid characters' do
+      out = inspec("init profile #{tmpdir}/inspecwith/slash")
+      out.exit_status.must_equal 0
```

Comment:
We should also check to see where the profile was created (two subdirs down) and what its name is (`slash` only).

Reply:
Thanks I'll resolve these!